flutter_vlc_player/lib/src/vlc_player_value.dart

```diff
@@ -134,6 +134,10 @@ class VlcPlayerValue {
     return aspectRatio;
   }
 
+   void setInitialized(bool initialized) {
+    _initialized = initialized;
+  }
+
```

Comment:
Not sure we want to expose this as a public method, since this class is exported to the plugin user. 

Making it a part of the `copyWith` would make the code a bit more consistent - it is used as an internal method for updating `VlcPlayerValue` fields. It's still part of the public interface, but using `copyWith` at least won't have any unexpected results in the code, since the user can't use the copied value anywhere in the plugin. 
At the same time updating the field on the player value can be read from elsewhere in the program, leading to undefined behavior.

Reply:
i agree with you.

Reply:
fixed

Reply:
Agree. these were side effects of removing part files. I was trying something to get the code to compile and pushed up for @alr2413. We probably should have worked in a side branch, when I pushed, I realized there was a risk that you would see it before we had a chance to talk about it.